surefire.css

```
@@ -0,0 +1,134 @@
+    font-size: 16px;
+    font-weight: 500;
+    color: #4552e8;
+    background-color: #ebedff;
+    border: solid 1px #cdd1ff;
+    padding: 10px 20px;
+}
+
+h2 {
+    padding: 10px 15px;
+    border: none;
+    border: dashed 2px #bcdfff;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)